config/routes.rb

```diff
@@ -3,7 +3,15 @@
 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
+  # get "up" => "rails/health#show", as: :rails_health_check
+  root "users#index"
+  # get "users/:user_id/posts" => "posts#index", as: :user_posts
+  # get "users/:user_id/posts/:id" => "posts#show", as: :user_post
+  # get "users" => "users#index", as: :users
+  # get "users/:id" => "users#show", as: :user
+  resources :users do
```

Comment:
As a requirement, kindly ensure you do not expose unnecessary endpoints. kindly ensure only the required routes are exposed. this protects your software from intruders or people with bad intentions😉.

<img width="766" alt="image" src="https://github.com/sadaf-Daneshgar/blogapp/assets/64643995/3ee985ae-fecc-43e0-aed5-d5dba746c555">


```suggestion
    resources :users, only: [:index, :show] do
```

Reply:
Hello @Ibizugbe , thank you for your nice review if I do like this it will be ok?
```
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
  end
```